core/src/main/java/org/infinispan/cache/impl/EncoderCache.java

```diff
@@ -860,4 +856,16 @@ public void addListener(Object listener) {
       initState(cache, this);
       return cache;
    }
+
+   private BiFunctionMapper wrapBiFunction(BiFunction<?, ?, ?> biFunction) {
```

Comment:
The compute/merge functions must be non-null according to `ConcurrentMap` javadoc.

Reply:
that's why I fixed. we were wrapping the null function and it only fails later in the code. now, it doesn't wrap and the superclass checks for null.

Reply:
I would like to keep it consistency with the remaining methods (put, replace, etc). The `keyToStorage()` and `valueToStorage()` pass `null` to the superclass too.